+incf/+livescripts/setup.m

```diff
@@ -60,11 +60,22 @@
             return;
         case "dandi"
             % Download the dataset
-            system("python 'example-live-scripts/+incf/+livescripts/dandi-downloader.py' " + dandiset_id + " /home/jovyan/");
+            path = ""
+            file = ""
+            if not(isempty(nwb_path))
+                [path, file, ext] = fileparts(nwb_path);
+                nwb_path = "/" + nwb_path;
+                file = file + ext;
+                path = dandiset_id + "/" + path + "/"
+            end
+            system("dandi download dandi://dandi/" + dandiset_id + nwb_path);
```

Comment:
Should we still try to use the dandi-downloader script?

Reply:
@rly, my bad, I totally forget to modify the python wrapper. Let me change that